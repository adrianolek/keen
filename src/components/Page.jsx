import React from 'react';
import PropTypes from 'prop-types';

const Page = ({title}) => (
  <div className="container-fluid">
    <div className="row">
      <div className="col-sm-12">
        <h2>{title}</h2>
      </div>
    </div>
  </div>
);

Page.propTypes = {
  title: PropTypes.string.isRequired
};

export default Page;
