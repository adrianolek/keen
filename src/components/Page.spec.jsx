import React from 'react';
import Page from './Page';
import renderer from 'react-test-renderer';

describe('Page', () => {
  it('renders with title', () => {
    const page = renderer.create(<Page title="Hello"/>).toJSON();
    expect(page).toMatchSnapshot();
  });
});
