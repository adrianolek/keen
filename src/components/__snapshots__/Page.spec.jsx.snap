// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page renders with title 1`] = `
<div
  className="container-fluid"
>
  <div
    className="row"
  >
    <div
      className="col-sm-12"
    >
      <h2>
        Hello
      </h2>
    </div>
  </div>
</div>
`;
